### PR TITLE
pgsql initial script sql

### PR DESCRIPTION
**Pull request by mckaygerhard**
pgsql initial script sql

### CODE REVIEW

File: SQL/pgsql.initial.sql [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+-- DROP TABLE userlogins;
+
+CREATE TABLE userlogins
+(
+  id serial NOT NULL,
+  "timestamp" timestamp without time zone NOT NULL DEFAULT now(),
+  user_id integer NOT NULL DEFAULT 0,
+  username character varying(128) NOT NULL DEFAULT VARCHAR(40),
+  sess_id character varying(128) NOT NULL DEFAULT VARCHAR(40),
```
Comment (mckaygerhard) [inline SQL/pgsql.initial.sql]: umm the sess_id i think must be 255 leng?

Comment (dsoares) [inline SQL/pgsql.initial.sql]: sess_id must be of the same length as the Roundcube session.sess_id.

**Comment by dsoares** [comment]
I added the `postgres.initial.sql` with a syntax similar to the same file of Roundcube install.

**Comment by mckaygerhard** [comment]
sorry for the sess_id, i made all of that in console with browser emulation and dont have any desktop the mayority of time a made some contribution

**Pull request closed without merging**